packages/video_player_videohole/tizen/src/drm_manager_service_proxy.cc

```diff
@@ -0,0 +1,145 @@
+}
+
+int DMGRGetData(void* handle, DRMSessionHandle_t drm_session,
+                const char* data_type, void** output_data) {
+  FuncDMGRGetData DMGRGetData;
+  *(void**)(&DMGRGetData) = Dlsym(handle, "DMGRGetData");
+  if (DMGRGetData) {
+    return DMGRGetData(drm_session, data_type, output_data);
+  }
+  return DM_ERROR_UNKOWN;
+}
+
+void DMGRSetDRMLocalMode(void* handle) {
+  FuncDMGRSetDRMLocalMode DMGRSetDRMLocalMode;
+  *(void**)(&DMGRSetDRMLocalMode) = Dlsym(handle, "DMGRSetDRMLocalMode");
+  if (DMGRSetDRMLocalMode) {
+    return DMGRSetDRMLocalMode();
+  }
+}
+
+DRMSessionHandle_t DMGRCreateDRMSession(void* handle, dm_type_e drm_type,
+                                        const char* drm_sub_type) {
+  FuncDMGRCreateDRMSession DMGRCreateDRMSession;
+  *(void**)(&DMGRCreateDRMSession) = Dlsym(handle, "DMGRCreateDRMSession");
+  if (DMGRCreateDRMSession) {
+    return DMGRCreateDRMSession(drm_type, drm_sub_type);
+  }
+  return nullptr;
+}
+
+FuncDMGRSecurityInitCompleteCB DMGRSecurityInitCompleteCB(void* handle) {
+  FuncDMGRSecurityInitCompleteCB DMGRSecurityInitCompleteCB;
+  *(void**)(&DMGRSecurityInitCompleteCB) =
+      Dlsym(handle, "DMGRSecurityInitCompleteCB");
+  if (DMGRSecurityInitCompleteCB) {
+    return DMGRSecurityInitCompleteCB;
+  }
+  return nullptr;
+}
+
+int DMGRReleaseDRMSession(void* handle, DRMSessionHandle_t drm_session) {
+  FuncDMGRReleaseDRMSession DMGRReleaseDRMSession;
+  *(void**)(&DMGRReleaseDRMSession) = Dlsym(handle, "DMGRReleaseDRMSession");
+  if (DMGRReleaseDRMSession) {
+    return DMGRReleaseDRMSession(drm_session);
+  }
+  return DM_ERROR_UNKOWN;
+}
+
+int player_set_drm_handle(void* handle, player_h player, player_drm_type_e drm_type,
+                               int drm_handle) {
+  FuncPlayerSetDrmHandle player_set_drm_handle;                         
+  *(void **)(&player_set_drm_handle) = Dlsym(handle, "player_set_drm_handle");  
+  if(player_set_drm_handle) {
+    return player_set_drm_handle(player, drm_type, drm_handle);
+  }                              
+  return 0;
+}
+
+int player_set_drm_init_complete_cb(void* handle, player_h player, 
+                                    security_init_complete_cb callback, 
+                                    void *user_data) {
+  FuncPlayerSetDrmInitCompleteCB player_set_drm_init_complete_cb;
+  *(void **)(&player_set_drm_init_complete_cb) = Dlsym(handle, "player_set_drm_init_complete_cb");                                    
+  if(player_set_drm_init_complete_cb){
+    return player_set_drm_init_complete_cb(player, callback, user_data);
+  }
+  return 0;
+}
+
+int player_set_drm_init_data_cb(void* handle, player_h player, 
+                                set_drm_init_data_cb callback, 
+                                void *user_data) {  
+  FuncPlayerSetDrmInitDataCB player_set_drm_init_data_cb;
+  *(void **)(&player_set_drm_init_data_cb) = dlsym(handle, "player_set_drm_init_data_cb");                                
```

Comment:
It's better to set variable name 'player_set_drm_init_complete_cb' different from function name.